src/impy/models/dpmjetIII.py

```diff
@@ -81,6 +82,19 @@ def __init__(self, event_kinematics, seed=None, logfname=None):
         dpm_conf = impy_config["dpmjetIII"]
 
         info(1, "Initializing DPMJET-III")
+
+        dpmjet_ver = pathlib.Path(dpm_conf["dat_dir"][self.version]).parts[-1]
+        if dpmjet_ver == "dpm3":
+            _cached_data_dir(
+                "https://github.com/impy-project/impy/releases/download"
+                "/zipped_data_v1.0/dpm3_v001.zip"
+            )
+        else:
+            _cached_data_dir(
+                "https://github.com/impy-project/impy/releases/download"
+                "/zipped_data_v1.0/dpm3191_v001.zip"
+            )
```

Comment:
If you need to check the version, you better use `self.version`.

`_cached_data_dir` should return the path of the data dir that we just downloaded here. This should be obsolete `pathlib.Path(dpm_conf["dat_dir"][self.version])`